test/danetest.c

```diff
@@ -287,25 +263,22 @@ static int tlsa_import_rr(SSL *ssl, const char *rrdata)
 
     for (f = tlsa_fields; f->var; ++f) {
         if ((len = f->parser(cp += len, f->var)) <= 0) {
-            fprintf(stderr, "%s: warning: bad TLSA %s field in: %s\n",
-                    progname, f->name, rrdata);
+            TEST_info("bad TLSA %s field in: %s", f->name, rrdata);
```

Comment:
and also, this prints info, but it returns failure which returns back to test_tlsafile, which returns failure which then reports an error in run_tlsatest.